src/common/types.ts

```diff
@@ -598,6 +598,18 @@ export interface GamesDBInfo {
   steamID: string
 }
 
+export interface ProtonCompatibility {
+  level: string
```

Comment:
Can you also fetch steam deck compatibility?

Reply:
Not from ProtonDB, so it'll be a different PR.

Reply:
Seems they not provide it via api yet but would be cool if they would soon. Is there somewhere documentation about the api?